webrtc/interfaces.html

```diff
@@ -0,0 +1,118 @@
+<!doctype html>
+<html>
```

Comment:
Got it. I was copying the boilerplate from the existing test files. Now the redundant markups are removed.